core/src/main/java/nl/nn/adapterframework/filesystem/FileSystemActor.java

```diff
@@ -577,7 +615,7 @@ public void setCreateFolder(boolean createFolder) {
 		this.createFolder = createFolder;
 	}
 
-	@IbisDoc({"4", "when set to <code>true</code>, for actions "+ACTION_MOVE+", "+ACTION_COPY+" or "+ACTION_RENAME+", the destination file is overwritten if it already exists", "false"})
+	@IbisDoc({"4", "when set to <code>true</code>, for actions MOVE, COPY or RENAME, the destination file is overwritten if it already exists", "false"})
```

Comment:
Would it be possible to use the refer to the values of the Enum constants here? E.g. by making them final?